src/nvim/generators/gen_lua_dispatch.lua

```diff
@@ -0,0 +1,324 @@
+-- we need at least 4 arguments since the last two are output files
+if arg[1] == '--help' then
+  print('Usage: genmsgpack.lua args')
```

Comment:
copy-pasta? ;)

Reply:
Yep, from `gen_api_dispatch.lua` from which it was also copy-pasta'd 😆